### ISSUE TITLE
result problem

### ISSUE THREAD

**Issue by makalo** [MegviiDetection/video_analyst#26]
I use the your training code to train. The training set only uses got10k. The test results are as follows
my   result:AO=0.612，SR0.5=0.735,SR0.75=0.461
your result:AO=0.595,SR0.5=0.695,SR0.75=0.479
Looks like performance is very good
But when I tested vot18 with this model, the result was very poor,The test results are as follows
my   result:ACC=0.549,ROB=0.300,EAO=0.310
your result:ACC=0.587,ROB=0.183,EAO=0.426
Do you have this problem

It is worth noting that the code for training is  "sh  tools/train_test-googlenet.sh" . When training is interrupted, I run "python ./main/train.py --config experiments/siamfcpp/train/siamfcpp_googlenet-trn.yaml --resume-from-file=... " to continue training

**Comment by lzx1413** [comment]
The model with EAO=0.426 is trained with multi-datasets like coco+youtubebb+lasot+got10k. We will release the multi-datasets training code in a few weeks, please keep tuned.

**Comment by makalo** [comment]
Can't support multi dataset training yet. I see  "names: ["GOT10kDataset",]  in the configuration file, but an error will be reported when I use  ["GOT10kDataset","LaSOTDataset"].

**Comment by lzx1413** [comment]
You are right. The training with multi-datasets is under testing yet, the original code is written with the internal framework. So the reproduce will take some time.

**Comment by songdejia** [comment]
> ce will take some t

Hi, can u provide the specific time of releasing the training code to reproduce the VOT18 and OTB100?

**Comment by lzx1413** [comment]
I can not guarantee one certain time but we will release it in this month hopefully.

**Comment by MARMOTatZJU** [comment]
@songdejia Training code is done and it is under final test (to make sure it works well after PyTorch version change) before being merged into Master branch. If you are urgent, you can check out the "dev-fulldata_hpo" branch in [my fork](https://github.com/MARMOTatZJU/video_analyst).

**Comment by songdejia** [comment]
> @songdejia Training code is done and it is under final test (to make sure it works well after PyTorch version change) before being merged into Master branch. If you are urgent, you can check out the "dev-fulldata_hpo" branch in [my fork](https://github.com/MARMOTatZJU/video_analyst).

Thanks. Can u provide a readme file to describe how to use full data

**Comment by ws20191229** [comment]
and in  fact you don't use youtube-BB dataset , you  use TrackingNet. there is  no  ytb_bb.py script in your program

**Comment by ws20191229** [comment]
@lzx1413 
We will release the multi-datasets training code in a few weeks, please keep tuned?  Did you have finished?  Thank  you !
